recipes-wolfssl/wolfssl/wolfssl_5.6.2.bb

```diff
@@ -14,12 +14,13 @@ RPROVIDES_${PN} = "cyassl"
 PROVIDES += "wolfssl"
 RPROVIDES_${PN} = "wolfssl"
 
-SRC_URI = "git://github.com/wolfssl/wolfssl.git;nobranch=1;protocol=https;rev=979707380c677dfa65e3ba48f19e149773a4a32d"
-SRC_URI += "file://6247.patch"
+DEPENDS += "util-linux-native"
+
+SRC_URI = "git://github.com/wolfSSL/wolfssl.git;branch=master;protocol=https;rev=9ffa9faecda87a7c0ce7521c83996c65d4e86943"
```

Comment:
I believe this file change is outdated now that Jacob updated us to `recipes-wolfssl/wolfssl/wolfssl_5.6.3.bb`.  Please rebase on master and adjust.

Reply:
I rebased, and removed `recipes-wolfssl/wolfssl/wolfssl_5.6.2.bb` and added `recipes-wolfssl/wolfssl/wolfssl_5.6.3.bb` from the upstream kirkstone branch